pkg/bundle/read.go

```diff
@@ -247,6 +251,76 @@ func read(ctx context.Context, name, baseDir string, bundleSpecReader io.Reader,
 	return New(def, scans...)
 }
 
+//ensureValidHelmRepo ensures that the Helm repo in the spec and all target customizations end with a '/', if any exist.
+//A '/' is appended if not. See chartURL
+func ensureValidHelmRepo(spec *fleet.BundleSpec) {
+	if spec.Helm != nil {
+		if spec.Helm.Repo != "" && !strings.HasSuffix(spec.Helm.Repo, "/") {
+			spec.Helm.Repo = spec.Helm.Repo + "/"
+		}
+	}
+
+	for _, target := range spec.Targets {
+		if target.Helm == nil || target.Helm.Repo == "" {
+			continue
+		}
+		if !strings.HasSuffix(target.Helm.Repo, "/") {
+			target.Helm.Repo = target.Helm.Repo + "/"
+		}
+	}
+}
+
+// propagateHelmChartProperties propagates root Helm chart properties to the child targets.
+func propagateHelmChartProperties(spec *fleet.BundleSpec, baseDir string) {
```

Comment:
This method should not be needed, because all three cases are handled by calculate.go already or did I miss something?

Reply:
The function is needed for the changes to be propagated to the downstream deployments.